src/pages/steps/AdditionalInformationStep.tsx

```diff
@@ -231,13 +253,68 @@ const AdditionalInformationStep = ({
                 onBlur={handleBlurDescription}
               />
             }
+            info={
+              <Stack spacing={3}>
+                {description.length < IDEAL_DESCRIPTION_LENGTH && (
+                  <ProgressBar
+                    variant={ProgressBarVariants.SUCCESS}
+                    progress={descriptionProgress}
+                  />
+                )}
+                <Text variant={TextVariants.MUTED}>
+                  {description.length < IDEAL_DESCRIPTION_LENGTH
+                    ? t(
+                        'create.additionalInformation.description.progress_info.not_complete',
+                        {
+                          idealLength: IDEAL_DESCRIPTION_LENGTH,
+                          count: IDEAL_DESCRIPTION_LENGTH - description.length,
+                        },
+                      )
+                    : t(
+                        'create.additionalInformation.description.progress_info.complete',
+                        {
+                          idealLength: IDEAL_DESCRIPTION_LENGTH,
+                        },
+                      )}
+                </Text>
+                <Button
+                  variant={ButtonVariants.LINK}
+                  onClick={handleClickClearDescription}
+                >
+                  {t('create.additionalInformation.description.clear')}
+                </Button>
+                {eventTypeId && (
+                  <Alert>
+                    <Box
+                      forwardedAs="div"
+                      dangerouslySetInnerHTML={{
+                        __html: t(
+                          `create*additionalInformation*description*tips*${eventTypeId}`,
+                          {
+                            keySeparator: '*',
+                          },
+                        ),
+                      }}
+                      css={`
+                        strong {
+                          font-weight: bold;
+                        }
+
+                        ul {
+                          list-style-type: disc;
+                          margin-bottom: ${parseSpacing(4)};
+
+                          li {
+                            margin-left: ${parseSpacing(5)};
+                          }
+                        }
+                      `}
+                    />
+                  </Alert>
+                )}
+              </Stack>
+            }
```

Comment:
```suggestion
            info={<DescriptionInfo />}
```

Maybe it would be best to seperate all this logic into its own component `DescriptionInfo`